src/__tests__/transfers.spec.ts

```diff
@@ -1,20 +1,24 @@
 import nock from "nock";
 import { createClient } from "../__mocks__/base";
 import { transfersSuccess, getTransactionSuccess, listTransactionsSuccess } from "../__mocks__/transfers/responses";
+import TransfersAPI from "../services/transfers";
 import Client from "../client";
-import { Transfers } from "../services";
-import { transfer} from "../typings";
+import {TransferInfo} from "../typings/transfers/transferInfo";
```

Comment:
Can we use the model collection import rather than importing the individual models?

Reply:
done